java_error_exceptions_block/src/main/java/Faculty.java

```diff
@@ -0,0 +1,15 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Faculty {
+    private String name;
+    public static List<Faculty> listOfFaculties = new ArrayList<>();
```

Comment:
please add comment or java doc for class which explains what static list stores

Reply:
done